pom.xml

```diff
@@ -84,5 +84,11 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
I also think this is not needed and I believe it came around because the Jetbrains package includes the following annotations:

> The @Nullable annotation indicates a variable, parameter, or return value that can be null.
> The @NotNull annotation indicates a variable, parameter, or return value that cannot be null.

```suggestion
```